frame/support/src/traits/misc.rs

```diff
@@ -425,6 +425,28 @@ impl<T> From<T> for WrapperOpaque<T> {
 	}
 }
 
+impl<T: MaxEncodedLen> MaxEncodedLen for WrapperOpaque<T> {
+	fn max_encoded_len() -> usize {
+		// Note: this can be improved if `T::max_encoded_len` is small.
+		// E.g. if T max encoded len is 4, then the compact encoding of its encoded length is 1.
```

Comment:
Why this comment here?

The point of this is to return the max encoded length?

Reply:
yes but I think we can be more precise in certain situation, I finally did the more precise implementation https://github.com/paritytech/substrate/pull/9881/commits/5f0718626783285161a28a87189ff586b4fe3b36